plugins/gpg/check-gpg-expiration.rb

```diff
@@ -106,8 +106,8 @@ def run
       critical
     when /^\d+$/
       message "Key #{config[:id]} has #{output} day(s) until it expires"
-      warning if output.to_i <= config[:warn] && output.to_i > config[:crit]
-      critical if output.to_i <= config[:crit]
+      warning if output.to_i <= config[:warn].to_i && output.to_i > config[:crit].to_i
```

Comment:
It is better to perform default type casting in `option` definition. Please take a look at example 

https://github.com/sensu/sensu-community-plugins/blob/master/plugins/haproxy/check-haproxy.rb#L69